package.json

```diff
@@ -17,6 +17,9 @@
     "onCommand:mint.init"
   ],
   "main": "./out/src/extension",
+  "dependencies": {
+    "vscode-languageclient": "^6.1.3"
```

Comment:
I don't know if it's needed or not, it was needed for testing the extension.

Reply:
Maybe let's move to the `devDependencies` then?

Reply:
Looks good.